crutest/src/main.rs

```diff
@@ -2005,9 +2112,31 @@ async fn replace_workload(
         }
     }
 
+    // The replace task may have already exited.
+    let _ = work_stop_tx.send(true).await;
+    if let Err(e) = handle.await.unwrap() {
+        bail!("Failed in IO workload: {:?}", e);
```

Comment:
I'm confused by the error message here – isn't `handle` the replace workload (and the IO workload is done in the loop right above, in this same task)?